### PR TITLE
[BANKCON-11876] Select first selectable account in Link Account Picker pane

### PR DESCRIPTION
**Pull request by mats-stripe**
## Summary

Fixing issue identified in [bug bash](https://docs.google.com/document/d/1MSzE2b2_nigERMxLkxtxdI4BETpACHoQ0UwMJ0i-Reg/edit?usp=sharing).

When showing the Link Account Picker pane, we now pre-select the first selectable account in the list. 

## Motivation

Consistency between platforms.

## Testing

| Before | After |
|--------|--------|
| Simulator Screenshot - iPhone 15 - 2024-08-14 at 11 12 57 | Simulator Screenshot - iPhone 15 - 2024-08-14 at 11 21 11 | 

## Changelog

N/a

### CODE REVIEW

File: StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift [hunk 1]
```diff
@@ -208,7 +208,11 @@ final class LinkAccountPickerViewController: UIViewController {
         self.footerView = footerView
         footerContainerView.addAndPinSubview(footerView)
 
-        bodyView.selectAccounts([]) // activate the logic to list all accounts
+        let firstSelectableAccount = accountTuples.first { accountTuple in
+            accountTuple.accountPickerAccount.allowSelection && accountTuple.accountPickerAccount.drawerOnSelection == nil
```
Comment (mats-stripe) [inline StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift:212]: [Android](https://github.com/stripe/stripe-android/blob/6c8953e7645b0b4e280fd9bc1a6a31e403683a19/financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt#L106)

**Review by bric-stripe — COMMENTED** [review]
does the button state need to be updated? in the after screenshot it looks like the button may be disabled, but not 100% sure

**Comment by mats-stripe** [comment]
> does the button state need to be updated? in the after screenshot it looks like the button may be disabled, but not 100% sure

Ahh yep, good catch! We should enable that button if an account gets pre-selected!

**Comment by mats-stripe** [comment]
@bric-stripe the button is now enabled when an account gets pre-selected! 

https://github.com/user-attachments/assets/4103ce9b-5612-43d3-bb6c-b9286aa6a9a3

**Review by bric-stripe — APPROVED** [review]
👍

**Pull request merged**